pyproject.toml

```diff
@@ -18,7 +18,9 @@ dependencies = [
     "deprecated",
     "ipykernel", # Support for Jupyter notebooks
     "biopython",
-    "streamlit"
+    "streamlit",
+    "python-Levenshtein", 
+    "pydantic"
```

Comment:
Could you specify the versions for these to ensure that there's no surprises?

Reply:
Yes, makes sense.